Misc/NEWS.d/next/Library/2024-12-03-20-28-08.gh-issue-127586.zgotYF.rst

```diff
@@ -0,0 +1,6 @@
+:class:`multiprocessing.pool.Pool` now properly restores blocked signal handlers
+of the parent thread when creating processes via either *spawn* or
+*forkserver*. Previously it would unblock **SIGTERM** and **SIGINT** in the
+parent thread on construction, even if either signal handler was
+intentionally blocked prior to the call to construct. Additionally, child
+processes can now inherit said blocked signals.
```

Comment:
Love the effort, but blurbs don't need to be nearly this long. Just the first sentence is fine :)
```suggestion
:class:`multiprocessing.pool.Pool` now properly restores blocked signal handlers
of the parent thread when creating processes via either *spawn* or
*forkserver*.
```